src/test/regress/expected/multi_extension.out

```diff
@@ -775,6 +775,13 @@ CREATE TABLE new_columnar_table (a int) USING columnar;
 ERROR:  loaded Citus library version differs from installed extension version
 -- do cleanup for the rest of the tests
 SET citus.enable_version_checks TO OFF;
+ALTER SYSTEM SET citus.enable_version_checks TO OFF;
```

Comment:
I tried removing the line above, i.e., `SET citus.enable_version_checks TO OFF;`, because now we alter this setting in system level however apperantly sometimes it doesn't take an affect on the current session. Any ideas @hanefi ?

Reply:
I get that the errors we saw was weird. There might be something we are failing to see here.

Although the current ALTER SYSTEM SET seems to work, I can not say that I like this solution.

Reply:
yeah I don't like it too